downstream/assemblies/navigator/assembly-review-inventory-navigator.adoc

```diff
@@ -0,0 +1,18 @@
+
+ifdef::context[:parent-context: {context}]
+
+
+[id="assembly-review-inventory-navigator_{context}"]
+= Review inventories with {Navigator}
+
+:context: review inventory-navigator
+
+[role="_abstract"]
+
+As a content developer, you can review your Ansible inventory with {Navigator} and interactively delve into the groups and hosts.
```

Comment:
just a nitpick we will have to change later across the board... instead of developer, say creator.  That's the official name we will be going for.

Reply:
okay I'll do that in a separate PR across all the navigator documentation files after this is merged. thanks!